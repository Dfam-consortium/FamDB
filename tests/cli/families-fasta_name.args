families
--format
fasta_name
-a
6
