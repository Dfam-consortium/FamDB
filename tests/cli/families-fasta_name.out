>Test family TEST0001 @root
ACGT
>Test family TEST0003 @Third_Clade [S:30,40]
GGTC
